blog/admin.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+
+from .models import User, Post, Commentary
+
+
+@admin.register(Post)
+class PostAdmin(admin.ModelAdmin):
+    list_display = ("title", "owner", )
+    search_fields = ("title", )
+
+
+@admin.register(Commentary)
+class CommentaryAdmin(admin.ModelAdmin):
+    list_display = ("created_time", "post", "user", )
+
+
+@admin.register(User)
+class UserAdmin(UserAdmin):

```

Comment:
There is a naming conflict with the `UserAdmin` class. Since you are importing `UserAdmin` from `django.contrib.auth.admin`, it would be better to rename your custom `UserAdmin` class to avoid confusion. Consider using a different name, such as `CustomUserAdmin`.